### ISSUE TITLE
iceprod2 - scheduled task - monitoring graphs (Trac #840)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#834]
Make monitoring graph statistics at scheduled intervals so we can build a historical profile.

Each site:
* Job states
* Task states

May need to separate out different queues on each site, if we get sites with multiple queues.

Possibly for each dataset as well?

Migrated from https://code.icecube.wisc.edu/ticket/840
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Make monitoring graph statistics at scheduled intervals so we can build a historical profile.\n\nEach site:\n* Job states\n* Task states\n\nMay need to separate out different queues on each site, if we get sites with multiple queues.\n\nPossibly for each dataset as well?\n", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - scheduled task - monitoring graphs", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-01-06T17:26:02", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150504T16:25:59: 
* david.schultz changed milestone from "" to "IceProd 2.1"
* david.schultz changed version from "2.0" to ""

**Comment by icecube-trac** [comment]
Trac update at 20160323T14:20:55: 
* david.schultz commented:

> Consider only providing basic daily graphs built-in, and using plugins to other tools like graphite and elastic serarch to do long-term monitoring.


* david.schultz changed milestone from "IceProd 2.1" to "IceProd 2.0"
* david.schultz changed priority from "normal" to "major"

**Comment by icecube-trac** [comment]
Trac update at 20160329T19:31:06: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 143881:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143881"
> basic task monitoring complete. closes #834
> ```


* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
